README.md

```diff
@@ -0,0 +1,49 @@
+Source for the frontend of R-hub builder
+========================================
+
+# Notes on how to build the website 
+
+## On macOS
+
+```
+npm install
+npm install -g  supervisor
+export JENKINS_URL=http://jenkins.r-hub.io:8080/
+export  LOGDB_URL=http://172.31.30.118:5984/logs
```

Comment:
Can you please change these to 
```
export JENKINS_URL=http://foo.bar
export LOGDB_URL=http://foo.bar
```
and now we also need 
```
MAILGUN_API_KEY=key
```

But I could actually just  make all these optional.... that's what  I'll do, including redis.

Reply:
so should I add "MAILGUN_API_KEY=key"?

Reply:
Yes, please. If you rebase you'll see that it is needed.